src/Uno.UWP/System/MemoryManager.Android.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Android.App;
+using Uno.UI;
+
+namespace Windows.System
+{
+	partial class MemoryManager
+	{
+		public static ulong AppMemoryUsage
+		{
+			get
+			{
+				return (ulong) GC.GetTotalMemory(false);
+			}
+		}
```

Comment:
If we are talking about Android, the "correct" way to do it (since it's a Unix OS) is by using `getrusage` instead. This method also considers shared memory, where the documentation of `MemoryManager.AppMemoryUsage` is not clear if it includes it or not.

Reply:
I think at this point shared memory is not very large great concern; we can stay on the java-android apis for now.